### PR TITLE
Graph typescript

### PR DESCRIPTION
**Pull request by milankoster**

### CODE REVIEW

File: src/components/Graph/GraphOptions.js [hunk 1]
```diff
@@ -1,4 +1,29 @@
...
+  return dataPointColour
+}
+
+// datasets for graph
+export function getGraphData(data) {
+  return {
+    datasets: [
+      {
+        data: data,
+        borderColor: graphColour,
```
Comment (vcaf) [inline src/components/Graph/GraphOptions.js:18]: small comment on different of using English and British English for color & colour

File: src/pages/PatientDashboard/PatientDashboard.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import Title from './Title/Title.tsx'
-import CommentList from './CommentList/CommentList.tsx'
-import Graph from '../../components/graph/graph'
+import Graph from '../../components/Graph/graph'
```
Comment (vcaf) [inline src/pages/PatientDashboard/PatientDashboard.tsx:1]: Could the graph file be started with Capital for consistency

File: src/components/Graph/graph.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import * as React from 'react'
+import 'chartjs-adapter-moment'
+import CommentModal from '../Modals/CommentModal/CommentModal'
+import { mockHr } from './MockData/hrData'
```
Comment (vcaf) [inline src/components/Graph/graph.tsx:4]: would starting with a capital be an option?

**Review by vcaf — APPROVED** [review]

File: src/components/Graph/graph.tsx (hunk 3 above)
Comment (milankoster) [reply src/components/Graph/graph.tsx:4]: Fixed :)

File: src/pages/PatientDashboard/PatientDashboard.tsx (hunk 2 above)
Comment (milankoster) [reply src/pages/PatientDashboard/PatientDashboard.tsx:1]: Fixed as well 👍

File: src/components/Graph/GraphOptions.js (hunk 1 above)
Comment (milankoster) [reply src/components/Graph/GraphOptions.js]: Changed to color, as GraphJS uses American English spelling

**Review by vcaf — APPROVED** [review]

**Pull request merged**